lib/modules/dosomething/dosomething_signup/dosomething_signup.module

```diff
@@ -562,12 +562,15 @@ function dosomething_signup_get_mbp_params($account, $node, $opt_in) {
   $url_options = array(
     'language' => $language,
     'absolute' => TRUE,
-    'prefix' => strtolower(dosomething_global_convert_language_to_country($language)) . '/',
     'query' => array(
       'source' => 'node/' . $node->nid,
     ),
   );
 
+  if ($country = strtolower(dosomething_global_convert_language_to_country($language))) {
```

Comment:
It will not always be true. If that function returns nothing, it will evaluate to false.